### PR TITLE
Warnings-Instance HTML/CSS, modified Warnings page hyperlinks

### PR DESCRIPTION
**Pull request by jenny-m-wood**
Screen Shot 2021-11-22 at 11 22 58 AM

### CODE REVIEW

**Comment by Pkoiralap** [comment]
LGTM

File: dashboard/static/css/warnings_style.css [hunk 1]
```diff
@@ -26,13 +26,15 @@ td {
...
   font-size: 30px;
   color: black;
   padding-bottom: 0.5%;
+  text-align: left;
 }
+.warnings_instance {
+  position: relative;
+  z-index: 0;
+  cursor: pointer;
+}
```
Comment (xhagrg) [inline dashboard/static/css/warnings_style.css]: New line.

File: dashboard/templates/warnings/WARNINGS_INSTANCE.html [hunk 2]
```diff
@@ -1,39 +1,47 @@
...
+		</div>
+		<div class="column">
+			<table>
+				<tr>
+					<th>Datasets Not in CDI Masterlist</th>
+				</tr>
+				{% for item in nimlist %}
+				<tr>
+					<td class="column_1_entry" onclick="window.location ='{{ item.catalog_url }}';">{{ item.title }}</td>
+				</tr>
```
Comment (xhagrg) [inline dashboard/templates/warnings/WARNINGS_INSTANCE.html]: Indent.

File: dashboard/templates/warnings/WARNINGS_INSTANCE.html [hunk 3]
```diff
@@ -1,39 +1,47 @@
...
+	<div class="row">
+		<div class="column">
+			<table>
+				<tr>
+					<th>Broken API Datasets</th>
+				</tr>
+				{% for item in brokenlist %}
+				<tr>
+					<td class="column_1_entry" onclick="window.location ='{{ item.catalog_url }}';">{{ item.title }}</td>
+				</tr>
```
Comment (xhagrg) [inline dashboard/templates/warnings/WARNINGS_INSTANCE.html]: Indent.

**Comment by andrewdweis** [comment]
Nice Work

**Pull request merged**